exercises/hangman/.meta/src/reference/java/Output.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * (C) ActiveViam 2019
+ * ALL RIGHTS RESERVED. This material is the CONFIDENTIAL and PROPRIETARY
+ * property of ActiveViam. Any unauthorized use
+ * reproduction or transfer of this material is strictly prohibited
+ */
```

Comment:
Because exercism is an open source project and contributors do their work voluntarily, we cannot accept PRs which have a copyright to the code.
Would you like to keep this?